docs/08-plugins/vault.md

```diff
@@ -0,0 +1,147 @@
+# vault  Plugin
+
+This plugin installs [vault](https://www.vaultproject.io/) in an existing Kubernetes cluster using the Helm chart for you test or develop vault.
+This plugin installs vault with replicas:3 by default value
+
+### Usage
+
+```yaml
+tools:
+- name: hashicorp-vault
+  # name of the plugin
+  instanceID: default
+  options:
+    # need to create the namespace or not, default: false
+    create_namespace: true
+    repo:
+      # name of the Helm repo
+      name: hashicorp
+      # url of the Helm repo
+      url: https://helm.releases.hashicorp.com
+    # Helm chart information
+    chart:
+      # name of the chart
+      chart_name: hashicorp/vault
+      # release name of the chart
+      release_name: vault
+      # k8s namespace where Vault will be installed
+      namespace: hashicorp
+      # whether to wait for the release to be deployed or not
+      wait: true
+      # the time to wait for any individual Kubernetes operation (like Jobs for hooks). This defaults to 5m0s
+      timeout: 5m
+      values_yaml: |
+        global:
+          enabled: true
+        server:
+          affinity: ""
+          ha:
+            enabled: true
+            replicas: 3
+            raft:
+              enabled: true
+              setNodeId: true
+          namespaceSelector:
+            matchLabels:
+              injection: enabled
+```
+
+Currently, except for `values_yaml`, all the parameters in the example above are mandatory.
```

Comment:
I do not believe this is true. For example, "timeout," "create_namespace" aren't mandatory.